org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/handler/ZigBeeThingHandler.java

```diff
@@ -364,7 +369,7 @@ private synchronized void doNodeInitialisation() {
                 }
 
                 logger.debug("{}: Initializing channel {} with {}", nodeIeeeAddress, channel.getUID(), handler);
-                if (handler.initializeDevice() == false) {
+                if (initializeDevice && handler.initializeDevice() == false) {
```

Comment:
Suggestion: Add a log statement somewhere that device initialization is skipped, because the device is already initialized. WDYT?